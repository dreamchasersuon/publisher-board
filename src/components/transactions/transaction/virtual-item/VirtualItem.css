.Transaction-item {
  display: flex;
  justify-content: space-between;
  margin-bottom: 5px;
}

.TransactionItem-name {
  font-size: 12px;
}

.TransactionItem-amount {
  font-size: 12px;
  color: #666;
}
