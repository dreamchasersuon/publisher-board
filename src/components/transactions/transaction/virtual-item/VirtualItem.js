import React from 'react';
import PropTypes from 'prop-types';
import './VirtualItem.css';

export default function VirtualItem({ item }) {
  return (
    <div className="Transaction-item">
      <span className="TransactionItem-name">
        {item.virtual_item.localized_name}
      </span>
      <span className="TransactionItem-amount">{item.amount}</span>
    </div>
  );
}

VirtualItem.propTypes = {
  item: PropTypes.object,
};
