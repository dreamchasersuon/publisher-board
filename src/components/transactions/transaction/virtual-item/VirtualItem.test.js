import React from 'react';
import renderer from 'react-test-renderer';
import VirtualItem from './VirtualItem';

test('renders purchased virtual item', () => {
  const item = {
    amount: 10,
    virtual_item: {
      sku: 22,
      localized_name: 'Snow Drum Machine',
    },
  };
  const tree = renderer.create(<VirtualItem item={item} />).toJSON();
  expect(tree).toMatchSnapshot();
});
