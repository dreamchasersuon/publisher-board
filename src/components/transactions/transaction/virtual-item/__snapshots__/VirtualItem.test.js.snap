// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders purchased virtual item 1`] = `
<div
  className="Transaction-item"
>
  <span
    className="TransactionItem-name"
  >
    Snow Drum Machine
  </span>
  <span
    className="TransactionItem-amount"
  >
    10
  </span>
</div>
`;
